client/model-config.json

```diff
@@ -68,5 +68,9 @@
   "InstanceAction": {
     "dataSource": "remote",
     "public": false
+  },
+  "Gateway": {
+    "dataSource": "remote",
+    "public": false
```

Comment:
this being the client side, the public field doesnt make any difference but Ive updated anyway